src/util.go

```diff
@@ -20,6 +20,7 @@ func init() {
 
 	CONFIG_DIRS = []string{
 		filepath.Join(home, ".tt"),
+		filepath.Join(home, ".config/tt"),
```

Comment:
Wouldn't `$XDG_CONFIG_HOME` be retrieved from the environment, and then use `.config/tt` as a fallback?
> If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used

Maybe something like this:
```go
xdg_config_home = os.Getenv("XDG_CONFIG_HOME")
if xdg_config_home == "" {
    xdg_config_home = filepath.Join(home, ".config")
}

CONFIG_DIRS = []string{
    filepath.Join(home, ".tt"),
    filepath.Join(xdg_config_home, "tt"),
    "/etc/tt",
}
```
(Disclaimer, this code is untested)